config/webpack.vendor.js

```diff
@@ -68,5 +67,5 @@ module.exports = [
   'jstz',
   'urijs',
   'uuid',
-  '@novnc/novnc/core/rfb.js',
+  // '@novnc/novnc/core/rfb.js',
```

Comment:
Does it relate to the PR scope?

Reply:
it should be removed.... @sharvit can you answer the following:
1. where should i put the loading section? I put it in index next to connect, not sure if we should split the two?
2. how do you suggest to do testings? have a look at the current failure (which I think might impact your answer on the first question)

thanks!

Reply:
Before you go ahead too far, I'd like to hear from @tbrisker too. I've seen clear -1 from him previously. Chances are that concerns were addressed or no longer apply.

@amirfefer, was the 100ms boost in dev or production environment? If in dev, can you also check impact on production?

Reply:
> * where should i put the loading section? I put it in index next to connect, not sure if we should split the two?
> * how do you suggest to do testings? have a look at the current failure (which I think might impact your answer on the first question)

I believe it belongs to the `index.js` while the `integration.test.js` should test the different loading states (pending, error, success).
Does the error appear because the `integration.test.js` doesn't wait for the loading?

Reply:
Another direction can be to wrap the `react-password-strength` lib  with Loadable

Reply:
@sharvit I did try both (in index and to wrap the npm package itself) in both cases the PasswordStrength/__tests__/integration.test.js should flow test failed

Reply:
I didn't get a chance to look into this PR too much, but IIRC my main concern in the past was around the change being focused on reducing download size, which is a one-time cost on production environments as all the assets should be cached and networks tend to be fast.
If this leads to a measurable improvement in page rendering speed in production than i don't have objections - as long as we have some benchmark results showing the improvement to be significant enough to be worth the extra effort and complexity. 
We should benchmark the difference both on pages that require the dynamically imported components and those that don't - I'd expect faster loads on pages that don't and slower results on pages that do (since they would need to wait for the extra file to be loaded and rendered before completing their own render), at which point we can say "we would rather wait 200ms more on pages X to make pages !X faster by 100ms"

Reply:
> @sharvit I did try both (in index and to wrap the npm package itself) in both cases the PasswordStrength/**tests**/integration.test.js should flow test failed

I believe it happens because of the `integration.test.js` doesn’t wait for the loading promise to get resolved.